### PR TITLE
update Rubocop and Travis config

### PR DESCRIPTION
**Pull request by jekyllbot**
PR automatically created for @DirtyF.

```text
update Rubocop and Travis config


- ran rubocop -a
- add remaining offenses to .rubocop.yml
- add script/fmt
- run script/fmt in script/cibuild
- update Travis rubies config
```

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,9 +1,15 @@
 inherit_gem:
   jekyll: .rubocop.yml
 
+Lint/ShadowingOuterLocalVariable:
+  Exclude:
+    - lib/jekyll-archives.rb
```
Comment (pathawks) [inline .rubocop.yml]: I probably want to fix this, but can do that in a separate PR

**Comment by DirtyF** [comment]
@jekyllbot: merge +dev
